Editor/Resources/EditorWindow/GameLiftPlugin.uxml

```diff
@@ -0,0 +1,36 @@
+<!-- Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. -->
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+<ui:UXML xmlns:ui="UnityEngine.UIElements" xmlns:uie="UnityEditor.UIElements" xsi="http://www.w3.org/2001/XMLSchema-instance" engine="UnityEngine.UIElements" editor="UnityEditor.UIElements" noNamespaceSchemaLocation="../../../../../UIElementsSchema/UIElements.xsd" editor-extension-mode="False">
+    <Style src="./GameLiftPlugin.uss" />
+    <ui:VisualElement name="Main" class="main">
+        <ui:VisualElement class="main__menu">
+            <ui:Button text="Amazon GameLift" name="Landing" class="tab__button" />
+            <ui:VisualElement class="Divider" />
+            <ui:Button text="AWS Account Credentials" name="Credentials" class="tab__button" />
+            <ui:VisualElement class="Divider" />
+            <ui:Button text="GameLift Anywhere" name="Anywhere" class="tab__button" />
+            <ui:VisualElement class="Divider" />
+            <ui:Button text="Managed EC2" name="ManagedEC2" class="tab__button" />
+            <ui:VisualElement class="Divider" />
+            <ui:Button text="Help &amp; Documentation" name="Help" class="tab__button" />
+        </ui:VisualElement>
+        <ui:ScrollView name="Content" horizontal-scroller-visibility="Hidden" class="main__content">
+            <ui:VisualElement name="LandingTab" class="tab__content">
```

Comment:
Instead of `*Tab` for names use `*Content` to be clearer since tab is used already to describe the buttons as well

Reply:
Updated

Reply:
Renamed